src/store/internals/removeEnabledElement.js

```diff
@@ -98,6 +100,17 @@ const _removeEnabledElement = function(enabledElement) {
   }
 };
 
+/**
+ * Remove load handler for the enabled element
+ * @private
+ * @method
+ * @param {HTMLElement} enabledElement
+ * @returns {void}
+ */
+const _removeLoadHandlers = function(element) {
+  loadHandlerManager.removeHandler(element);
```

Comment:
This needs to be updated to `removeHandlers` as well.

Reply:
done